cypress/e2e/Navbar.cy.js

```diff
@@ -0,0 +1,46 @@
+describe('Test the Navabr routing works', () => {
+  it('Successfully navigates to the Home Page upon clicking Home', () => {
+    cy.visit('http://localhost:3000/');
+    cy.get('[data-cy="home"]').click();
+    cy.url().should('include', '/');
+  });
+  it('Successfully navigates to the About Page upon clicking About', () => {
+    cy.visit('http://localhost:3000/');
+    cy.get('[data-cy="about"]').click();
+    cy.url().should('include', '/About');
+  });
+  it('Successfully navigates to the Contact Page upon clicking Contact', () => {
+    cy.visit('http://localhost:3000/');
+    cy.get('[data-cy="contact"]').click();
+    cy.url().should('include', '/Contact');
+  });
+
+  it('Successfully navigates to the FAQ Page upon clicking FAQ', () => {
```

Comment:
Same for all these tests. Start the test names with should. 

E.g should successfully navigate to the FAQ page upon clicking FAQ